clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -74,9 +74,18 @@ static droption_t<std::string>
               "Specifies the file path of the PT raw trace. Please run the "
               "libipt/script/perf-read-aux.bash script to get PT raw trace file from the "
               "data generated by the perf record command.");
+static droption_t<std::string>
+    op_elf(DROPTION_SCOPE_FRONTEND, "elf", "", "[Optional] Path of the elf file",
+           "Specifies the file path of the elf file. You must specify when "
+           "converting short traces that don't contain sideband information.");
+static droption_t<unsigned long long>
+    op_elf_base(DROPTION_SCOPE_FRONTEND, "elf_base", 0x0,
+                "[Optional] The base address of the elf file",
+                "Specifies the base address of the elf file.");
```

Comment:
I believe this must be specified if --elf is specified. If so, add it to the description.

Reply:
The elf_base option is a optional in elf mode. How about I change the optional to ELF Mode Optional?

Reply:
Done.